src/analyses/guard.h

```diff
@@ -12,9 +12,20 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_ANALYSES_GUARD_H
 #define CPROVER_ANALYSES_GUARD_H
 
+#ifdef BDD_GUARDS
+
+#include "guard_bdd.h"
+
+using guard_managert = guard_bdd_managert;
+using guardt = guard_bddt;
```

Comment:
Would it be possible for them to please have a shared interface? Otherwise inconsistencies will creep in that can only be detected by compiling both with and without `BDD_GUARDS` set.

Reply:
I suppose we had the same discussion on the CUDD/miniBDD BDD PR :-)  It's not the most important of all issues and can be left to a future PR - but I will remind you about it whenever there is a change to these interfaces and someone forgets to update both of them :-P

Reply:
fair enough